test/java/seedu/addressbook/commands/CountCommandTest.java

```diff
@@ -0,0 +1,52 @@
+package seedu.addressbook.commands;
+
+import org.junit.Test;
+import org.junit.Assert;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.data.person.*;
+
+import java.util.HashSet;
+
+public class CountCommandTest {
+
+    AddressBook testAddressBook = new AddressBook();
+
+    @Test
+    public void testCountCommandOutput() {
```

Comment:
Do look at the name of the tests for other commands. This is not really following the naming convention of the tests.